WordPress/Classes/Utility/DiskCache.swift

```diff
@@ -10,6 +10,8 @@ import CryptoKit
 /// - important: It's possible to have more than one instance of ``DiskCache`` with
 /// the same path but it is not recommended.
 public actor DiskCache {
+    public static let shared = DiskCache(name: "AppDiskCache")
```

Comment:
It has to be a singleton assuming you want to limit its maximum, which I think is generally a good idea and should be built in, or at least designed in such a way that it would be possible to build in in the future.

Reply:
Using a singleton so that we can apply a storage limit feels like a big compromise to me. I think I'd be more comfortable if users can choose to use the shared instance or create a dedicated instance for their specific feature.